### PR TITLE
migrate to AndroidX

### PR DESCRIPTION
**Pull request by dulmandakh**
Migrated source code to use AndroidX, and Example to use React Native 0.60.5. Also removed unnecessary files like flow and buck configurations, Gradle wrapper in android, etc... Removed Geolocation module from Example on iOS.

I can compile for Android and iOS versions of the example app, but metro fails to resolve react-native-reanimation from '../src/Animated' via babel configuration.

**HELP**: Help make metro work for example app.

### CODE REVIEW

**Comment by dulmandakh** [comment]
@kmagiera please review and merge

**Comment by dulmandakh** [comment]
Duplicate PRs:  https://github.com/kmagiera/react-native-reanimated/pull/383, https://github.com/kmagiera/react-native-reanimated/pull/340, https://github.com/kmagiera/react-native-reanimated/pull/334.
Related issues: https://github.com/kmagiera/react-native-reanimated/issues/374, https://github.com/kmagiera/react-native-reanimated/issues/337, https://github.com/kmagiera/react-native-reanimated/issues/290, https://github.com/kmagiera/react-native-reanimated/issues/288

File: Example/metro.config.js [hunk 1]
```diff
@@ -1,21 +1,17 @@
...
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ */
 
 module.exports = {
-  resolver: {
-    blacklistRE: getBlacklist(),
-    providesModuleNodeModules: ['react-native', 'react', 'fbjs'],
+  transformer: {
```
Comment (kmagiera) [inline Example/metro.config.js]: I'd prefer if we keep this config. It have us a way to import source from reanimated directly from top level directory which helps a lot with development

**Comment by dulmandakh** [comment]
It wasn't working either

File: Example/metro.config.js (hunk 1 above)
Comment (dulmandakh) [reply Example/metro.config.js]: I can revert the change, but it's not working either

Comment (kmagiera) [reply Example/metro.config.js]: I think this setup no longer works with metro from 0.61 and @osdnk had some workaround for it. Lets revert for now or sync up on getting that workaround pulled in

**Comment by dulmandakh** [comment]
@kmagiera reverted metro.config.js to original version.

**Comment by kmagiera** [comment]
cool, thank you

**Comment by kmagiera** [comment]
We will fix metro config in follow up PR

**Comment by dulmandakh** [comment]
Please close duplicate PRs and issues 👍

**Comment by SaeedZhiany** [comment]
@kmagiera 

it seems #399 should also be closed.

**Pull request merged**